gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenStateService.java

```diff
@@ -160,7 +156,8 @@ public void revokeToken(final JWTToken token) {
   @Override
   public void revokeToken(final String token) {
     validateToken(token);
```

Comment:
If removeToken() invokes validateToken(), then we can skip the validateToken() call here.